auslib/test/test_db.py

```diff
@@ -1109,6 +1113,11 @@ def testEnactChangeExistingRow(self):
         self.assertEquals(history_row.data_version, 2)
         self.assertEquals(sc_row.complete, True)
 
+    def testEnactChangeForDeletingExistingRow(self):
```

Comment:
Great work on the tests so far. Just one more to add still: something that tests mergeUpdate for a scheduled deletion.